### DISCUSSION TITLE
There is no way to display empty metadata fields in filter webpart

### DISCUSSION THREAD

**Discussion by amilanc** [microsoft-search/pnp-modern-search#3380] [Q&A]
I have couple of questions.
1.	This is a question on pnp search filter. I use several refinable managed properties as filters in the search page.  Is there a way to get the filter to show empty metadata fields. Currently the filter only shows ones with data. We are planning on using this as a tool to let users know that they need to fill required metadata. What I meant is having the <Blank> as an option in filter criteria
2.	In the search results page, how can I add a link to the item properties so users can see the metadata.
3.	In the search results page, how can I add a link to the library where a result is displayed

**Comment by kasperbolarsen** [comment]
Hi

Ad 1) sorry, it is not possible 
Ad 2) not sure as I not know if the results are list items or library items, but one option could be to create a hover card showing the metadata 
Ad 3) the Library URL most likely is in the results,  check it using SharePoint search query tool or SP Editor
